mobile/src/screens/meditation-list/meditation-list.tsx

```diff
@@ -40,6 +39,28 @@ const MeditationList: React.FC = () => {
     'title',
   );
 
+  const [isModalVisible, setIsModalVisible] = useState(false);
+  const [duration, setDuration] = useState(DEFAULT_SONG_DURATION);
+
+  const handleClick = (): void => {
+    setIsModalVisible(!isModalVisible);
+  };
+
+  const handleClose = (): void => {
+    setIsModalVisible(!isModalVisible);
+  };
+
```

Comment:
```suggestion

  const handleToggleClick= (): void => {
    setIsModalVisible((previous) => {
      return !previous;
    });
  };

```
Is it possible to make it in one function? Something like handleToggleClick?

Reply:
@OlenaYatseniuk good solution )